src/main/resources/entity-types/circulation/simple_loan.json5

```diff
@@ -368,5 +368,16 @@
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->'reminders'->'lastFeeBilled'->>'date'",
     },
+    {
+      name: 'loan_jsonb',
```

Comment:
Is there a reason we can't just call each column 'jsonb' (instead of prepending it with the entity type name)? None of the other columns use the prepending, and the translations aren't using it either.